### PR TITLE
TEST-001_loginfunctionalitydone

### PR DESCRIPTION
**Pull request by bajajm208**
Hi Ankit, I have done the login functionality

### CODE REVIEW

File: src/main/java/com/policybazar/pagelayer/loginpage.java [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package com.policybazar.pagelayer;
+
+public class loginpage {
+
+	public static void main(String[] args) {
```
Comment (ankitbajaj66) [inline src/main/java/com/policybazar/pagelayer/loginpage.java:5]: Don't use main method

File: src/main/java/com/policybazar/pagelayer/loginpage.java [hunk 2]
```diff
@@ -0,0 +1,10 @@
+package com.policybazar.pagelayer;
+
+public class loginpage {
```
Comment (ankitbajaj66) [inline src/main/java/com/policybazar/pagelayer/loginpage.java:3]: not good

**Review by ankitbajaj66 — CHANGES_REQUESTED** [review]

File: src/main/java/com/policybazar/pagelayer/loginpage.java (hunk 1 above)
Comment (bajajm208) [reply src/main/java/com/policybazar/pagelayer/loginpage.java:5]: done

File: src/main/java/com/policybazar/pagelayer/loginpage.java (hunk 2 above)
Comment (bajajm208) [reply src/main/java/com/policybazar/pagelayer/loginpage.java:3]: done

**Pull request merged**